openhtf/io/frontend/__main__.py

```diff
@@ -100,12 +90,14 @@ def Refresh(self):
                        self.hostport)
         self.station_id = station_id
         self.state = state
+        result = True
```

Comment:
just return True here, since it's the only path that sets it to true, and put a return False at the bottom